### PR TITLE
[MH-69]  accelerate development

### PR DESCRIPTION
**Pull request by kmtracey**
Strip out most everything except code we are actually using now.

### CODE REVIEW

**Comment by dsummersl** [comment]
Can one of the admins verify this patch?

File: hydroshare/urls.py [hunk 1]
```diff
@@ -1,200 +1,22 @@
...
 
-# Adds ``STATIC_URL`` to the context of error pages, so that error
-# pages can use JS, CSS and images.
-handler404 = "mezzanine.core.views.page_not_found"
-handler500 = "mezzanine.core.views.server_error"
+# HMMM....? Shouldn't these be served by nginx for debug False?
+if settings.DEBUG is False:   # if DEBUG is True it will be served automatically
+    urlpatterns += [
+        url(r'^static/(?P<path>.*)$', 'django.views.static.serve', {'document_root': settings.STATIC_ROOT}),
+]
```
Comment (dsummersl) [inline hydroshare/urls.py]: I would think so...and it looks like it does:

nginx/config-files/hydroshare-nginx.conf#17

Comment (dsummersl) [reply hydroshare/urls.py]: (there are two nginx configuration templates - one for SSL one w/o...but they both appear to serve /static)

Comment (kmtracey) [reply hydroshare/urls.py]: OK...so I guess this must be for the local dev case of having DEBUG=False which I now recall was the default until I did something to turn it on. I guess we could leave it (I realize it looks like I added it, really I just moved and properly indented it) and clarify the commenting...

**Review by dsummersl — CHANGES_REQUESTED** [review]
I know this isn't caused by your changes, but it is a small change: as you mentioned in chat the `ChooseNetworkTestCase.test_POST_custom_provider` test case is failing.

From what I can tell this test started failing when the 'health networks' migrations were introduced. Because the test sets up a user associated with NC, and NC has health networks associated with it, the test is not redirecting. The easiest fix I see is to associate the user with some other state. I dropped this into the test case to make use of the unused `state` variable (although I'd rather a query be made to explicitly pick a state with no health networks rather than trusting that the first one is healthcare-network-free):

```
        self.user.userdetails.state = state
        self.user.userdetails.save()
```

**Comment by kmtracey** [comment]
screenshot from 2018-07-13 14-35-12

**Review by dsummersl — APPROVED** [review]

**Pull request merged**